NMS-12228: document additional expandable event parameters. (#2651)
OpenNMS/opennms: opennms-doc/guide-admin/src/asciidoc/text/events/event-configuration.adoc

    The operator instructions for the event.
`%mouseovertext%`::
    The mouse over text for the event.
`%tticketid%`::
    The trouble ticket id associated with the event, if available.
`%primaryinterface%`::
The primary interface IP address for the node given in `%nodeid%` if available.

.Asset tokens

> A node may have additional asset meta data stored for it. You can access that metadata using the `asset` replacement token, which takes the form:

Reviewer: ```suggestion
A node may have additional asset records stored for it.
You can access these records using the `asset` replacement token, which takes the form:
```

asset meta-data -> asset records to help avoid being confuse with the new "node meta-data"
one sentence per line

`%asset[<token>]%`::
    The asset field <token>'s value, or "Unknown" if it does not exist.

.Hardware tokens

> A node may have additional hardware meta data stored for it. You can access that metadata using the `hardware` replacement token, which takes the form:

Reviewer: ```suggestion
A node may have additional hardware details stored for it.
You can access these details using the `hardware` replacement token, which takes the form:
```

similar as above
